content_scripts/desktopCamera.js

```diff
@@ -269,6 +269,61 @@ createNameSpace('realityEditor.device.desktopCamera');
             document.body.appendChild(closestObjectLog);
         }
 
+        const keyboard = new realityEditor.device.KeyboardListener();
+
+        // Setup Save/Load Camera Position System
+        // Allows for quickly jumping between different camera positions
+        let getSavedCameraDataLocalStorageKey = (index) => `savedCameraData${index}-${realityEditor.sceneGraph.getWorldId()}`;
+        
+        const saveCameraData = (index) => {
+            const cameraPosition = [...virtualCamera.position];
+            const cameraDirection = virtualCamera.getCameraDirection();
+            const cameraData = { cameraPosition, cameraDirection };
+            const cameraDataJsonString = JSON.stringify(cameraData);
+            localStorage.setItem(getSavedCameraDataLocalStorageKey(index), cameraDataJsonString);
+        }
+        
+        const loadCameraData = (index) => {
+            const cameraDataJsonString = localStorage.getItem(getSavedCameraDataLocalStorageKey(index));
+            if (!cameraDataJsonString) {
+                return;
+            }
+            try {
+                const cameraData = JSON.parse(cameraDataJsonString);
+                virtualCamera.position = [...cameraData.cameraPosition];
+                virtualCamera.setCameraDirection(cameraData.cameraDirection);
+                return cameraData;
+            } catch (e) {
+                console.warn('Error parsing saved camera position data', e);
+            }
+        }
+        
+        // Only one gets a menu item to avoid crowding, but they all get a shortcut key
+        const saveCameraPositionMenuItem = new realityEditor.gui.MenuItem('Save Camera Position', { shortcutKey: '_1', modifiers: ['ALT'], toggle: false, disabled: false }, () => {
+            saveCameraData(0);
+            realityEditor.gui.getMenuBar().getItemByName('Load Camera Position').enable();
+        });
+        const loadCameraPositionMenuItem = new realityEditor.gui.MenuItem('Load Camera Position', { shortcutKey: '_1', modifiers: ['SHIFT'], toggle: false, disabled: loadCameraData(0) === null }, () => {
```

Comment:
Small bug I noticed: `loadCameraData(0)` is undefined, not null, when you haven't saved a position yet, so the `=== null` is false and the menu item isn't disabled